site/_docs/reference.md

```diff
@@ -3132,6 +3133,10 @@ parameters:
   |   identifier
 {% endhighlight %}
 
+| C | Operator syntax                                | Description
+|:- |:-----------------------------------------------|:-----------
+| s | EXISTS(array, predicate)                       | Returns whether a *predicate* holds for one or more elements in the *array*
+
```

Comment:
I don't think you need this table twice in the same file. I think you should replace this table with a paragraph 'Examples of functions with a lambda argument are EXISTS, ...'.